src/features/page_objects/land_boundary/image-file-check.page.js

```diff
@@ -0,0 +1,17 @@
+const Page = require("../page");
+
+class LandBoundaryCheckImageFile extends Page {
+  get titleText() {
+    return "Check the land boundary file";
+  }
+
+  get path() {
+    return "/land/check-land-boundary";
+  }
+
+  open() {
+    return super.open("land/check-land-boundary");
+    // return super.open("land/check-land-boundary-file");
```

Comment:
can this be removed?

Reply:
design haven't finalised the naming convention yet... will tidy up when they do